packages/common/src/types.ts

```diff
@@ -8,11 +8,11 @@ export interface genesisStatesType {
   [key: string]: {}
 }
 
+export interface ChainName {
+  [chainId: string]: string
+}
 export interface chainsType {
```

Comment:
Not sure, seems this was lower capital before, but this should be actually capitalized. Can you do along?